### DISCUSSION TITLE
Atom Program Explanation

### DISCUSSION THREAD

**Discussion by Jelizog** [merkourisa/Augmented-Reality#7] [Q&A]
(That's only for users that downloaded the program "Atom")

1. To use Atom is a must to make it a default program using the URI files?
2. Must we Install a Package or this is for other purposes?
3. Will the Snippets make our environment easier? 
4. What file ought to insert inside? (Is the one from GitHub desktop?) or are we creating a new one?
5. What's the average size of the file (.URI), So we could possibly save it in a disk that we know has plenty of space to save and test inside there.

P.S. I'll write more questions if there are some in the near future

**Comment by merkourisa** [comment]
1. Αtom and Brachets are IDEs that you can use to edit javascript and .html files. You can use also use Visual Studio Code that is installed in the university labs or you can use a simple Notepad to edit the code.
4. You don't have to create a new file, the code is already given to you in Github. You just have to make the appropriate changes. You have to download the files (with Github desktop or Git Gui) to your Pc. Edit the files with the IDE, test them locally with the webserver and finally upload the program in Github (with Github desktop or Git Gui). In deliverable 1, I explain which file you have to edit. Nevertheless, I will a make demonstration on Friday during lab hours.
5. the .html files are very small. The assets (3D models) that you will use will be larger.

You are welcome to ask any questions. I am eager to  see more students participating in the discussions
